test/unit/spec/common/cellComponents/tabs/jobStateList-Spec.js

```diff
@@ -1,76 +1,98 @@
 define([
-    'base/js/namespace',
     'common/cellComponents/tabs/jobStatus/jobStateList',
-    'jquery',
     'common/props',
     'json!../../../../../data/testAppObj.json',
-], (Jupyter, jobStateList, $, Props, TestAppObject) => {
+], (jobStateList, Props, TestAppObject) => {
     'use strict';
 
-    describe('The job status tab module', () => {
+    let cssBaseClass;
+
+    const model = Props.make({
+        data: TestAppObject,
+        onUpdate: () => {},
+    });
+
+    function createInstance() {
+        return jobStateList.make({
+            model: model,
+        });
+    }
+
+    describe('The job state list module', () => {
         it('loads', () => {
             expect(jobStateList).not.toBe(null);
         });
 
         it('has expected functions', () => {
             expect(jobStateList.make).toBeDefined();
         });
-    });
 
-    describe('The job status tab instance', () => {
-        beforeAll(() => {
-            Jupyter.narrative = {
-                getAuthToken: () => 'fakeToken',
-            };
+        it('has a cssBaseClass variable', () => {
+            expect(jobStateList.cssBaseClass).not.toBeNull;
+            expect(jobStateList.cssBaseClass).toContain('kb-job');
         });
+    });
 
-        afterAll(() => {
-            Jupyter.narrative = null;
-        });
-
-        let node, model, mockJobStateList;
-
+    describe('The job state list instance', () => {
+        let jobStateListInstance;
```

Comment:
renamed `mockJobStateList` to `jobStateListInstance`, since it is more appropriate